src/main.rs

```diff
@@ -277,14 +278,24 @@ mod tests {
         } else {
             Path::new("tests/empty-base")
         };
+        // Empty dir, needed so that no warnings are printed when testing older Nix versions
+        // that don't recognise certain newer keys in nix.conf
+        let nix_conf_dir = tempdir()?;
```

Comment:
A cute way to do this would be to take a dependency on `pkgs.emptyDirectory`.

Reply:
Agreed, but since we're not in Nix here, it would be a bit cumbersome, so I think it's fine as-is :P

Reply:
Totally.